examples/formik/js/async-validator/helper.css

```diff
@@ -0,0 +1,91 @@
+* {
```

Comment:
Remove it. * has not specifity

Reply:
Is just the example from formik official site, I would keep it like it is, Maybe we should open a PR to them :),